### PR TITLE
upgrade go-genproto submodules to avoid ambiguous import errors

### PR DESCRIPTION
**Pull request by paupm**
Followed this comment: https://github.com/googleapis/go-genproto/issues/1015

To try and fix the errors I got in the previous build: https://github.com/softonic/homing-pigeon/actions/runs/10347140488/job/28636856541

When creating a grpc connection pool, DialOptions like WithBlock and the `grpc.Dial` are bad practices and were deprecated. More info: https://github.com/grpc/grpc-go/blob/master/Documentation/anti-patterns.md

### CODE REVIEW

**Comment by bvis** [comment]
This is needed for continue with this task: https://softonic-development.atlassian.net/issues/DS-1907

**Review by mindhells — APPROVED** [review]
Have no idea what I'm approving 😅

File: pkg/main.go [hunk 1]
```diff
@@ -19,9 +20,8 @@ func main() {
 
 	bufLen = GetBufferLength("ACK_BUFFER_LENGTH")
 	ackCh1 := make(chan messages.Ack, bufLen)
```
Comment (joskfg) [inline pkg/main.go:22]: 1 is not needed :-)

**Review by joskfg — APPROVED** [review]

**Comment by paupm** [comment]
> This is needed for continue with this task: https://softonic-development.atlassian.net/issues/DS-1907

Actually this PR is just to upgrade Go version and get rid of Dingo.
Let's align if we need to work on allowing middlewares between writer and reader flow.

**Pull request merged**